Packs/Slack/Scripts/SlackBlockBuilder/SlackBlockBuilder.yml

```diff
@@ -41,17 +41,25 @@ args:
   defaultValue: NoResponse
 - description: The instance of SlackV3 this script should use.
   name: slackInstance
+- description: The ID of the thread to which to reply. Can be retrieved from a previous send-notification command.
+  name: thread_id
 outputs:
 - contextPath: SlackBlockState
   type: unknown
   description: The state of the response from the user will be stored under this context path.
+- contextPath: SlackBlockBuilder
+  description: The thread ID retrieved from the message notification
```

Comment:
Please update the full path of each returned field in the output
You can use the following [guide](https://xsoar.pan.dev/docs/integrations/context-and-outputs#:~:text=Copy-,Integration%20YML%23,-In%20the%20integration)